src/jira/test.js

```diff
@@ -0,0 +1,33 @@
+const config = require("../config");
+const notifyConfig  = config.get("notify");
+var JiraClient = require('jira-connector');
+
+var jira = new JiraClient( {
```

Comment:
This has been defined previously, so it would be better to move this to a modules export `/jira/config.js` and the import it from that file and use it. This avoids duplication and variations.

Reply:
Added a new config file, and imported; much neater, thanks @DesignyourCode